src/MeetingsSDKAdapter.test.js

```diff
@@ -179,47 +173,37 @@ describe('Meetings SDK Adapter', () => {
       meetingSDKAdapter.getStream(meetingID, {sendAudio: true})
         .pipe(last())
         .subscribe((localMedia) => {
-          expect(localMedia).toMatchObject({
-            stream: mockSDKMediaStreams.localAudio,
-            permission: 'ALLOWED',
-          });
+          expect(localMedia.stream).toMatchMediaStream(mockSDKMediaStreams.localAudio);
+
```

Comment:
you can lose this empty line, there is a single `expect` of each type in this case